compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -327,3 +327,4 @@ error.intervening.whitespaces.are.not.allowed=intervening whitespaces are not al
 error.invalid.binding.pattern=invalid binding pattern
 error.resource.path.cannot.begin.with.slash=resource path cannot begin with slash
 error.missing.field.match.pattern.member=missing field match pattern member
+error.included.record.parameter.cannot.be.rest.parameter=included record parameter cannot be a rest parameter
```

Comment:
Since we parse `*Configuration ...config` as `INVALID[*] Configuration ...config`, parsed node will be a rest parameter node. 

Therefore, the diagnosis should be something like `'*' is not allowed for rest parameter`

Reply:
discussed offline and changed diagnostic as `rest parameter cannot be an included record parameter`